.gitignore

```diff
@@ -153,6 +146,17 @@ npm-debug.log*
 node_modules
 static/
 
+# Hash files
 data/**/*.md5
 
+# Evaluation
+app/backend/evaluation/report
```

Comment:
Is anything ever generated in there? AFAICT, unless we explicitly specify the path, this should go under the timestamped experiment directories.

Reply:
This is only used in some local machine tests, but its okay to remove that